README.md

```diff
@@ -1,220 +1,130 @@
+Please refer to the [PPA release page][ppa] and the [installation documentation](doc/INSTALL.md) for detailed instructions.
 
-It has the following environment variable:
-
-* `EXPOSE_TCP` default to false, if true, use expose c-lightning RPC on port 9835. (Use this only for testing)
-
-Here is an example of a docker-compose file with bitcoind and c-lightning on `testnet` which expose bitcoind's RPC interface on default ports `18332` and c-lightning API on port `9735`:
+For the impatient here's the gist of it for Ubuntu:
 
-```
-version: "3"
-services:
-  bitcoind:
-    image: nicolasdorier/docker-bitcoin:0.16.3
-    container_name: bitcoind
-    environment:
-      BITCOIN_EXTRA_ARGS: |
-        testnet=1
-        whitelist=0.0.0.0/0
-        server=1
-        rpcuser=rpcuser
-        rpcpassword=rpcpass
-    expose:
-      - "18332"
-    ports:
-      - "0.0.0.0:18333:18333"
-    volumes:
-      - "bitcoin_datadir:/data"
-
-  clightning_bitcoin:
-    image: elementsproject/lightningd
-    container_name: lightningd
-    command:
-      - --bitcoin-rpcconnect=bitcoind
-      - --bitcoin-rpcuser=rpcuser
-      - --bitcoin-rpcpassword=rpcpass
-      - --network=testnet
-      - --alias=myawesomenode
-      - --log-level=debug
-    environment:
-      EXPOSE_TCP: "true"
-    expose:
-      - "9735"
-    ports:
-      - "0.0.0.0:9735:9735"
-    volumes:
-      - "clightning_bitcoin_datadir:/root/.lightning"
-      - "bitcoin_datadir:/etc/bitcoin"
-    links:
-      - bitcoind
-
-volumes:
-  bitcoin_datadir:
-  clightning_bitcoin_datadir:
+```bash
+sudo apt-get install -y software-properties-common
+sudo add-apt-repository -u ppa:bitcoin/bitcoin
+sudo add-apt-repository -u ppa:lightningnetwork/ppa
+sudo apt-get install bitcoind lightningd
 ```
 
 ### Starting `lightningd`
 
-In order to start `lightningd` you will need to have a local `bitcoind`
-node running in either testnet or regtest mode:
+In order to start `lightningd` you will need to have a local `bitcoind` node running (in this case we start `testnet`):
 
-    bitcoind -daemon -testnet
+```bash
+bitcoind -daemon -testnet
+```
 
 Wait until `bitcoind` has synchronized with the testnet network.
 
-Make sure that you do not have `walletbroadcast=0` in your
-`~/.bitcoin/bitcoin.conf`, or you may run into trouble.
-Notice that currently pruned nodes are not supported and may result in
-`lightningd` being unable to synchronize with the blockchain.
+Make sure that you do not have `walletbroadcast=0` in your `~/.bitcoin/bitcoin.conf`, or you may run into trouble.
+Notice that running `lightningd` against a pruned node may cause some issues if not managed carefully, see [below](#pruning) for more information.
 
 You can start `lightningd` with the following command:
 
-    lightningd/lightningd --network=testnet --log-level=debug
+```bash
+lightningd --network=testnet --log-level=debug
+```
+
+Please refer to `lightningd --help` for all other command line options.
```

Comment:
we might want to explain the config-dir variable or at least mention where `~/.lightning`-dir will be placed by default.